src/main/java/org/jabref/gui/preferences/externalfiletypes/EditExternalFileTypeEntryDialog.java

```diff
@@ -36,9 +36,12 @@ public class EditExternalFileTypeEntryDialog extends BaseDialog<Void> {
 
     private final ExternalFileTypeItemViewModel item;
 
+    private final boolean isNewItem;
     private EditExternalFileTypeViewModel viewModel;
 
     public EditExternalFileTypeEntryDialog(ExternalFileTypeItemViewModel item, String dialogTitle) {
+        this.isNewItem = (item.extensionProperty().get().equals("")) ? true : false;
```

Comment:
No need for mapping true to true

```suggestion
        this.isNewItem = (item.extensionProperty().get().equals(""));
```